woocommerce/woocommerce-android
# issue/3431-dark-done-button-5.8

Fixes #3431 - To test:

* Go to product detail
* Add linked products
* Verify the Done button (see below) is light in both light mode and dark mode

Note that I changed the Done button into a checkmark, which is a familiar pattern in action mode. Note also that I made a new checkmark drawable for this to ensure the check icon is colored correctly regardless of SDK version (`iconTint` isn't available < SDK 26).

![light](https://user-images.githubusercontent.com/3903757/104649677-151f2800-5683-11eb-88a7-be4d8437d080.png)

Update release notes:
